yt_dlp/extractor/tv8.py

```diff
@@ -0,0 +1,71 @@
+import re
+
+from yt_dlp.utils import str_or_none, traverse_obj
+
```

Comment:
yt-dlp currently uses relative imports
```suggestion
from .common import InfoExtractor
from .skyit import TV8ItIE
from ..utils import clean_html, url_or_none, urljoin
```